pkg/controller/operators/catalog/subscription/reconciler.go

```diff
@@ -96,6 +125,11 @@ func (c *catalogHealthReconciler) catalogHealth(namespace string) ([]v1alpha1.Su
 		catalogs = append(catalogs, globals...)
 	}
 
+	// Sort to ensure ordering
+	sort.Slice(catalogs, func(i, j int) bool {
```

Comment:
we only really need this for testing, right? is there any way we can just sort for tests and not for prod?